content/post/thriving-oss-community.md

```diff
@@ -0,0 +1,138 @@
+ 
+If you are interested in speaking to us about how Wallaroo could help solve your particular problems, you can jump the queue and schedule a call with me directly by [clicking here](https://calendly.com/chuckb/wallaroo-user-interview).
+ 
+You can see that maintaining availability is really important to us. That’s why [we’re hiring a full-time developer evangelist](http://careers.wallaroolabs.com/apply/99yvBVfMGM/Developer-Evangelist) to contact Wallaroo developers, identify use cases, and get early feedback on the product.  We want to get information in front of developers and conduct as many of these user interviews as possible!  
+ 
+## Talk to Users
+
+In addition to user interviews, we are continually attending conferences and meetups to build awareness and find the right audience for Wallaroo.
+ 
+Even when we aren’t presenting, attending these events and speaking to developers directly is a great way to get feedback. Face-to-face interactions often help us build a better understanding of the problems and challenges that developers face.
+ 
+## Listen to feedback
+
+We are proud of Wallaroo and have put blood, sweat, and tears into the product.  Naturally, it can be difficult to hear criticism.
+ 
+Our job is to make sure that we listen to feedback, both the good and the bad.  Some of the most useful information can be found in the seemingly hard criticism.
+ 
+After a [recent blog post](https://blog.wallaroolabs.com/2017/10/go-python-go-stream-processing-for-python/) on our Python API, [we received some comments](https://news.ycombinator.com/item?id=15457343) that our API needed to be "more Pythonic." We listened to that feedback, spoke to many users directly, and incorporated their ideas into our roadmap for the Python API (updated version coming later this year).
+ 
+If we closed our ears to the criticism of our users, we would be betraying all the work and feedback they gave us.
+ 
+## Conclusion
+ 
+We believe that building a great user community is possible if you set measurable goals, are honest about your code, treat your community with respect, and engage them often. The community we have built around Wallaroo has helped us get closer to our long-term goal of making this the best tool that developers building distributed data processing applications can possibly get.
+ 
+These are just some of the ways that we are going about it, and we expect to incorporate even more ideas into our process as we continue to hear your feedback and learn from our mistakes. 
+
+Let us know what you think would bring you into our community by sharing this post with your feedback on social media.
+ 
+__As I mentioned earlier, we are currently looking to hire a full-time Developer Evangelist in the SF Bay Area to help us continue to build our community.  If you are interested, you can [check out the job posting here](http://careers.wallaroolabs.com/apply/99yvBVfMGM/Developer-Evangelist).__
```

Comment:
It doesn't look like text between `__` gets rendered differently on the blog than any other text. I'm not sure if this is a Hugo issue or our style or what, but if we want this text to jump out we might want to try something else.

Reply:
I think it dislikes that when there's a link. I'm going to remove the __